index.js

```diff
@@ -97,7 +97,7 @@ const credentialsManager = new CredentialsManager(logger, argv.awsRegion, argv['
     await trash(paths.data);
   }
 
-  if (!argv.headful) {
+  if (!argv.headful && !argv.listAvailableRoles) {
```

Comment:
Without adding this, the output looks something like this:

```
❯ node index.js --idp-id=redacted --sp-id=redacted --aws-region us-west-2 --list-available-roles
⠋ Logging inarn:aws:iam::1234567890:role/StagingAccount-AdministratorAccess
arn:aws:iam::1234567890:role/StagingAccount-FullReadOnlyAccess
arn:aws:iam::1234567890:role/StagingAccount-ReadOnlyAccess
arn:aws:iam::1234567891:role/ProductionAccount-AdministratorAccess
arn:aws:iam::1234567891:role/ProductionAccount-FullReadOnlyAccess
arn:aws:iam::1234567891:role/ProductionAccount-ReadOnlyAccess
```

I have to filter out stderr to get the output to look good

```
❯ node index.js --idp-id=redacted --sp-id=redacted --aws-region us-west-2 --list-available-roles 2>/dev/null
arn:aws:iam::1234567890:role/StagingAccount-AdministratorAccess
arn:aws:iam::1234567890:role/StagingAccount-FullReadOnlyAccess
arn:aws:iam::1234567890:role/StagingAccount-ReadOnlyAccess
arn:aws:iam::1234567891:role/ProductionAccount-AdministratorAccess
arn:aws:iam::1234567891:role/ProductionAccount-FullReadOnlyAccess
arn:aws:iam::1234567891:role/ProductionAccount-ReadOnlyAccess
```

Looking for recommendation on what's the best way to address this.